Bai-tap-Function-2/Bai4.py

```diff
@@ -0,0 +1,22 @@
+'''Bài 4: Viết hàm nhận vào n => In ra hình tam giác như Ví dụ.
+
+Input: func4(4)
+Output:
+****
+***
+**
+*
+'''
+
+def print_tri(m): 
+    x = m
+    arr = []
+    while x<=m and x>0: 
+        arr.append(x)
+        x -= 1
+    for i in arr:
+        print("*" * i,end ='')
+        print()
+
+
+print_tri(6) 
```

Comment:
@ngochan21098 Please add a new line.

Reply:
@ddhienpy I have fixed it here
https://github.com/ngochan21098/learning-python-basic/commit/4f0786b56802b21ec8c79670e2c95a79be05bde1